setup.py

```
@@ -29,7 +29,7 @@ def _get_single_requirement(requirements: List[str], package: str) -> List[str]:
 class BuildPyCommand(build_py):
     def run(self):
         try:
-            self.run_command('compile_contracts')
+            self.run_command('verify_contracts')
```

What's the point of running this here if the exception is silently swallowed?